Scripts/NuGet/EdFi.Installer.WebApi/Install-EdFiOdsWebApi.psm1

```diff
@@ -160,7 +160,11 @@ function Install-EdFiOdsWebApi {
         # NuGet package version. If not set, will retrieve the latest full release package.
         [string]
         $PackageVersion,
-        
+
+        # NuGet package source . default value is set for release package source for installer .
+        [string]
+        $PackageSource = "https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi%40Release/nuget/v3/index.json",
```

Comment:
Passing in the local feed here fails when it tries to list the package version inside the `Get-NuGetPackage` function. The function assumes that only one package will be listed as a result of `nuget list`, but since it searches for a substring of package name it picks up both EdFi.Suite3.Ods.WebApi, and EdFi.Suite3.Ods.WebApi.Prerelease. This isn't a problem for our release feed because it only contains EdFi.Suite3.Ods.WebApi.

Nuget install will automatically install the latest version if no `version` parameter is provided, so the following code can likely be removed:
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/blob/b2ac2cf0e831aa5443115fafce3f969ccf357e5c/logistics/scripts/modules/packaging/nuget-helper.psm1#L105-L115

Also I think we can remove the check to see if it is already downloaded since `nuget install` does this by default.
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/blob/b2ac2cf0e831aa5443115fafce3f969ccf357e5c/logistics/scripts/modules/packaging/nuget-helper.psm1#L117-L123

then change the parameters to only pass in `PackageVersion` if it is not null (this is marked as an optional parameter):
```powershell
$parameters = @(
    "install", $PackageName,
    "-source", $PackageSource,
    "-outputDirectory", $OutputDirectory
)
if ($PackageVersion) {
    $parameters += "-version"
    $parameters += $PackageVersion
}
```
and finally modifying the returned value to resolve without a version using wildcards:
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/blob/b2ac2cf0e831aa5443115fafce3f969ccf357e5c/logistics/scripts/modules/packaging/nuget-helper.psm1#L135